src/util/system.h

```diff
@@ -115,6 +115,18 @@ inline bool IsSwitchChar(char c)
 #endif
 }
 
+struct DefaultArgHints {
+    constexpr DefaultArgHints() = default;
```

Comment:
Any reason to have the default constructor and specify default values for the default args? Note that the compiler forces initialization when a member is `const`, so moving the initialization of the defaults to bitcoind.cpp seems cleaner.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19471#issuecomment-655603970).